tests/test_integration.py

```diff
@@ -144,6 +148,27 @@ def test_worker_service_called_multiple_times(self):
         eq_(payload['backend'], {'app': 'django', 'app_type': 'web'})
         eq_(payload['database'], {'app': 'postgres', 'app_type': 'db'})
 
+    @skipUnless(False,
+        'test to try the http log errors, to trigger it change'
+        + 'https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/api.py#L78'
+        + 'to "conn.request("HEAD", endpoint, data, self._headers)"'
+        + 'skip TestAPITransport and TestAPIDowngrade as they will fail if you change PUT to HEAD')
```

Comment:
i think more important than testing the log capture itself is testing that the timestamps you are using for tracking errors are correctly updated. this means forcing an error in the api, and ensuring that `last_error_ts` is updated to a reasonable value.